.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.testing.pytestArgs": [],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
This file isn't really part of the repo, so should be excluded. The best way would probably be in a global gitignore for `.vscode` directories, e.g. https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.

Reply:
I think I ran the global gitignore, but would like to confirm.